guides/manual-installation.md

```diff
@@ -65,7 +65,7 @@ systemctl enable --now prometheus
 Allow prometheus to be accessible from docker. Add an exception on firewalld:
 
 ```bash
-firewall-cmd --zone=public --add-port=9090/tcp --permanent
+firewall-cmd --zone=public --add-port=9090/tcp --permanent;
```

Comment:
The `;` doesn't hurt but it's not needed: if a user copy-pastes two lines, the newline char will also get copied and cause that two separate commands are executed.